### ISSUE TITLE
What's the difference between "X_ = torch.cat((X_,X_tmp), dim=1)" and "X_ = torch.cat([X_,X_tmp], dim=1)"

### ISSUE THREAD

**Issue by why986** [seongjunyun/Graph_Transformer_Networks#19]
At first I use the latter to train my model and get about 88 f-score on test dataset(ACM), then I change it to the former and get about 92 f-score. What's the difference?

**Comment by why986** [comment]
Also, when I use the former,  I sometimes get nan loss...

**Comment by seongjunyun** [comment]
Hi,
sorry for the late reply.

As I know, there is no difference. The difference in performances is maybe due to random seed.
If you get nan loss, I suggest you change the learning rate or the weight decay. 

Thank you.
